experimental/sql/src/ibis_frontend.py

```diff
@@ -36,7 +30,7 @@ def convert_datatype(type_: ibis.expr.datatypes) -> id.DataType:
   raise KeyError(f"Unknown datatype: {type(type_)}")
 
 
-def visit_schema(schema: ibis.expr.schema.Schema) -> Region:  #type: ignore
+def visit_schema(schema: ibis.expr.schema.Schema) -> Region:
```

Comment:
Why does this function and the next not use multipledispatch? Add a small comment?

Reply:
They return Regions as they work on multiple nodes in the ibis tree whereas the other functions return Operations for the single tree nodes they work. Added a comment :)